tests/time_tests/test_runner/conftest.py

```diff
@@ -87,6 +88,15 @@ def pytest_addoption(parser):
         help='collection name in "{}" database'.format(DATABASE),
         choices=DB_COLLECTIONS
     )
+    db_args_parser.addoption(
+        '--db_metadata',
+        type=str,
+        help='add extra commit information, json formatted')
+    db_args_parser.addoption(
+        '--manifest',
```

Comment:
OS version can't be parsed from manifest, so need to provide it within --db_metadata

Reply:
agree, my bad